index.html

```diff
@@ -26,41 +33,320 @@
+              <div class="language-container">
+                <p class="language-text">html</p>
+              </div>
+            </li>
+            <li class="language-item">
+              <div class="language-container">
+                <p class="language-text">css</p>
+              </div>
+            </li>
+            <li class="language-item">
+              <div class="language-container">
+                <p class="language-text">javaScript</p>
+              </div>
+            </li>
+          </ul>
+        </div>
+        <div class="card-footer">
+          <button type="button" class="card-button">See Project</button>
+        </div>
+      </figure>
+    </section>
+    <section id="about-me">
+      <div class="intro-about">
+        <h2 class="about-header">About<br />Myself</h2>
+        <p class="about-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hesitate to
+          contact me.
+        </p>
+        <a href="#" class="link" id="second-link"
+          ><p class="connect">LET’S CONNECT</p></a
+        >
+        <ul class="social-media" id="second-social-media">
+          <li>
+            <a href="#"
+              ><img src="./utils/Vector.svg" alt="twitter" class="twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img
+                src="./utils/Linkedin-icon.svg"
+                alt="linkedin"
+                class="linkedin"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="./utils/Vector1.svg" alt="elipse" class="medium"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="./utils/Vector2.svg" alt="elipse" class="github"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="./utils/Vector3.svg" alt="elipse" class="angel"
+            /></a>
+          </li>
+        </ul>
+        <div class="card-footer">
+          <button type="button" class="card-button" id="about-button">
+            Get my resume
+          </button>
+        </div>
+      </div>
+    </section>
+    <section id="dropdown-menu">
+      <ul class="base-list">
+        <li class="list-item" id="languages">
+          <div class="list-item-inserted">
+            Languages <i class="fas fa-chevron-right fa-rotate-90"></i>
+          </div>
+          <div>
+            <ul class="language-lists">
+              <li class="language-item-in">
+                <div class="language-list-container">
+                  <img
+                    src="./utils/js.svg"
+                    alt="javascript"
+                    class="language-item-logo"
+                  />
+                  <p class="language-item-text">JavaScript</p>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") being a list should be ordered using `ul` and each of them a `li`.

Reply:
I insert language codes into the ul and li tags.

Reply:
:+1: This issue has been addressed successfully :heavy_check_mark: